doc/user_guide/user_guide.md

```diff
@@ -1066,3 +1066,53 @@ changes.
 [delta-io]: https://delta.io/
 [delta-storage]: https://docs.delta.io/latest/delta-storage.html
 [delta-history]: https://docs.delta.io/latest/delta-utility.html#history
+
+## Hadoop Distributed Filesystem (HDFS)
+
+The [Hadoop distributed file system (HDFS)]() is a distributed, scalable, and
+portable file system written in Java for the Hadoop framework
+
+When the Hadoop datanodes and Exasol cluster are installed in the same (virtual)
+network, you can access the HDFS using `cloud-storage-extension`.
+
+For import:
+
+```sql
+IMPORT INTO <schema>.<table>
+FROM SCRIPT CLOUD_STORAGE_EXTENSION.IMPORT_PATH WITH
+  BUCKET_PATH     = 'alluxio://<ALLUXIO_PATH>/import/orc/data/*.orc'
```

Comment:
Is aluxio correct here? Isent't this section about hdfs

Reply:
Fixed.